### DISCUSSION TITLE
Can I run multi validators with one beacon-chain and one execution node?

### DISCUSSION THREAD

**Discussion by yasyzb** [OffchainLabs/prysm#13808] [General]
I have run a validator + beacon chain + execution node. Now I want to add a new validator node  without restart the old one.
Can I run another validator node with a new machine and connect to the old beacon chain node?

**Comment by nisdas** [comment]
@yasyzb Yes, that is perfectly fine

**Reply by yasyzb** [reply]
Thank you
